### DISCUSSION TITLE
Parallel plugin routing in the same ag instance

### DISCUSSION THREAD

**Discussion by Ardalien** [apohl79/audiogridder#409] [Ideas]
Hello. Is it possible to add in the audiogrider plugin a gain/pan and the possibility to route the plugins in parallel ?. A lot of plugins have a dry wet knob but It would be good if You could control the routings of the chains in the audiogridder plugin without needing to add to add multiple ag instances. The latency builds a lot up when adding multiple ag chains in series :p. It would be a great solution for complex projects to be able to mix a signal chain routings with one or two parallel chains of plugins inside one audiogridder instance so you don t have to add multiple ag’s. I guess it will be more efficient to use less I/O ports between the client and the server to free up the network bandwith in bigger projects.

**Comment by Ardalien** [comment]
Well, a lot of the plugins have gain control so that is not a problem but as they load one after another in series why not be able to split the line and add two in parallel after one in series and more.

**Comment by apohl79** [comment]
Thanks for your suggestion. I see where you are coming from. My current view on this though is, that i'd like to keep AG being a network bridge basically, that does not touch the audio data itself. I will will create a feature request for this eventually, but I would prioritize other topics around usability and stability right now. Hope that makes sense!

**Reply by Ardalien** [reply]
Cool ! . I believe you, there still are more things to be done about stability and usability. I just thought that as the audio passes in series from one plug to another It might not be very difficult to adjust a parallel routing :P. But I don't really understand the architecture of it.. I hope you can pull this of and it will be amazing and extremely useful for then adding lesser instances. Good luck, wish you the best !

**Reply by apohl79** [reply]
Thanks!
